.golangci.yml

```diff
@@ -97,4 +97,4 @@ issues:
     text: "cyclomatic complexity 34 of func `openstackValidationFunc` is high"
 
   - path: pkg/apis/kubeone
-    text: "cyclomatic complexity 31 of func `ValidateCloudProviderSpec` is high (> 30)"
+    text: "cyclomatic complexity 31 of func `ValidateCloudProviderSpec` is high"
```

Comment:
Can we fix this instead of ignoring the linter error?

Reply:
I can take care of this later with some other PR :P